02-molecules/cards/cards.stories.js

```diff
@@ -43,3 +44,17 @@ export const cardWithBackground = () => (
     nibh pulvinar a.
   </Card>
 );
+export const newsCard = () => (
+  <NewsCard
+    image={{
+      alt: 'Random Image from Picsum',
+      src: 'https://picsum.photos/400/300.jpg',
```

Comment:
Want to match the Twig demo site? It uses src "https://picsum.photos/1280/720".

Reply:
yep